catalog/security/catalog-security-plugin/src/main/java/ddf/catalog/security/plugin/SecurityPlugin.java

```diff
@@ -51,10 +52,10 @@ public CreateRequest processPreCreate(CreateRequest input) throws StopProcessing
           .getMetacards()
           .forEach(
               metacard -> {
-                if (metacard.getTags().isEmpty() || metacard.getTags().contains("resource")) {
-                  metacard.setAttribute(
-                      new AttributeImpl(
-                          Metacard.POINT_OF_CONTACT, SubjectUtils.getEmailAddress(subject)));
+                String emailAddress = SubjectUtils.getEmailAddress(subject);
```

Comment:
Is there a reason why we wouldn't want to use the unique identifier here as well?